lepidopter-vmdebootstrap_build.sh

```diff
@@ -12,8 +12,6 @@ APT_MIRROR="http://httpredir.debian.org/debian"
 MIRROR="http://httpredir.debian.org/debian"
 TODAY=`date +%Y%m%d-%H%M`
 
-export APT_MIRROR
```

Comment:
Correct. The parameter ```-a``` of set command marks variables which are modified or created for export.